<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyWidget</class>
 <widget class="QWidget" name="KeyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <spacer name="verticalSpacer_14">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p1_forward:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p1_rotate_left:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p1_rotate_right:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p1_shoot:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p1_shield:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>end_the_game:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_13">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>suspend_replay:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p2_forward:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p2_rotate_left:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p2_rotate_right:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p2_shoot:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>p2_shield:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_14">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>decelerate_replay:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_15">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
         </property>
         <property name="text">
          <string>accelerate_replay:%1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_16">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	margin : 3px;
	padding : 10px;
	color : white;
	text-align : left;
	background-color : rgba(70, 70, 70, 65%);
}
QPushButton:checked{
	background-color : rgba(0, 0, 0, 55%);
}
QPushButton:pressed{
	background-color : rgba(0, 0, 0, 55%);
}</string>
       </property>
       <property name="text">
        <string>game model %1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_12">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_13">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
